run.py

```diff
@@ -107,7 +108,15 @@ def main():
                 if not test_file.endswith(".js"):
                     continue
 
-            stats = run_benchmark(executable, executable_arguments, suite, test_file, score_metric, args.iterations, current_test, total_tests)
+            current_test += 1
```

Comment:
Maybe move this in a `finally` block? Not sure if it's used for anything aside from printing logs, nvm if not.

Reply:
I've simply moved it up and set the initial value to 0, so the value during run_benchmark() is identical

Reply:
ah gotcha, I missed that.
all good then!